### PR TITLE
#164722615 Add environment variable for swagger url

### PR DESCRIPTION
**Pull request by IssaIan**
### What does this PR do?
- Changes the swagger url to an environment variable
### How to manually test 
- In your .env, add a variable `SWAGGER_URL` with the address of the server of your running application. *e.g.* `https://questioners-two-staging.herokuapp.com/`
### Relevant Pivotal Tracker Stories?
[#164722615](https://www.pivotaltracker.com/n/projects/2315729/stories/164722615)

### CODE REVIEW

File: api/config/swagger.py [hunk 1]
```diff
@@ -15,7 +16,7 @@
 def schema_view_swagger(request):
     schema = coreapi.Document(
         title='Questioner API',
-        url='https://questioners-two-staging.herokuapp.com/',  # replace with your url
+        url=os.getenv('swagger_url'),
```
Comment (Ogutu-Brian) [inline api/config/swagger.py]: Hello @IssaIan ,
By convention, environment variables are usually defined using all capital letters. This helps users distinguish environmental variables within other contexts.
You can change the variable `sawagger_url` to `SWAGGER_URL` to meet that convention.

**Review by Ogutu-Brian — CHANGES_REQUESTED** [review]

File: api/config/swagger.py (hunk 1 above)
Comment (IssaIan) [reply api/config/swagger.py]: Implemented

**Review by Ogutu-Brian — APPROVED** [review]

**Review by Amoswachira — APPROVED** [review]

**Review by Bakley — COMMENTED** [review]
The PR Title is not Andela Convention, please change that

File: api/config/swagger.py [hunk 2]
```diff
@@ -197,7 +198,7 @@ def schema_view_swagger(request):
                 ),
                 'logout': coreapi.Link(
                     url='/api/auth/logout/',
-                    action='DELETE',
+                    action='POST',
```
Comment (davenjeru) [inline api/config/swagger.py]: This is not part of the task @IssaIan

**Review by davenjeru — CHANGES_REQUESTED** [review]

**Pull request merged**